sources/packages/api/src/route-controllers/program-info-request/models/program-info-request.dto.ts

```diff
@@ -1,15 +1,22 @@
-import { IsNotEmpty, IsInt, Min, IsOptional } from "class-validator";
-import { SaveOfferingDTO } from "../../education-program-offering/models/education-program-offering.dto";
+import {
+  IsNotEmpty,
+  IsInt,
+  Min,
+  IsOptional,
+  IsPositive,
+} from "class-validator";
+import { SaveOfferingDTO } from "src/route-controllers/education-program-offering/models/education-program-offering.dto";
 import { ProgramInfoStatus } from "../../../database/entities";
 import { OfferingIntensity } from "../../../database/entities/offering-intensity.type";
 
-export interface CompleteProgramInfoRequestDto extends SaveOfferingDTO {
-  selectedProgram?: number;
-  selectedOffering?: number;
+export class CompleteProgramInfoRequestAPIInDTO {
+  @IsPositive()
+  selectedOffering: number;
 }
 
-export interface GetProgramInfoRequestDto
-  extends CompleteProgramInfoRequestDto {
+export interface GetProgramInfoRequestDto extends SaveOfferingDTO {
```

Comment:
I believe that this DTO can be reviewed also and does not need to be extended from `SaveOfferingDTO`.
Can we please have the DTO renamed to ProgramInfoRequestAPOutDTO and have the properties reviewed and added to the DTO itself?

Reply:
I do not see this one changed or a reply to it. Is it going to be changed?

Reply:
No need to do it right now, we can leave for the API refactor phase.